controllers/storagecluster/initialization_reconciler_test.go

```diff
@@ -49,7 +50,8 @@ func createStorageCluster(scName, failureDomainName string,
 	return cr
 }
 
-func createUpdateRuntimeObjects(cp *Platform) []runtime.Object {
+func createUpdateRuntimeObjects(cp *Platform, namespace string, c client.Client) []runtime.Object {
```

Comment:
Rather than passing in the parameters required for `avoidObjectStore()`, see if you can make a separate call to `avoidObjectStore()` outside this function and just pass in the boolean.

While you're at it, go ahead and group all the objects behind an `if avoidObjectStore()` stanza into one longer `if` statement. We shouldn't need to get the same boolean four times if the value is never going to change. However, please make this change its own commit.